cni-plugin/integration/Dockerfile-tester

```diff
@@ -9,6 +9,7 @@
 #  2) a test suite (e.g. `flannel.go`) runs with a configured CNI plugin.
 
 FROM golang:1.18-alpine AS build
+RUN apk add build-base
```

Comment:
Were we missing tooling from alpine-go base image?

Reply:
When I added `mapstructure`, the container failed with `Cgo` errors which I tracked down to needing something provided by `build-base`, gcc I think.